### PR TITLE
Fix #61

### LINKED ISSUE #61: [HTML] Fail on <pre> tags
Hi.
This tool seems to be working pretty flawlessly, good job.
I noticed just a case on which it fails: when trying to minify HTML code, if `<pre>` tags are present, it fails.

Waiting for updates.

### PR DESCRIPTION
**Pull request by bamless**
Fix issue #61 by skipping only the \<pre>\</pre> and <textarea></textarea> blocks during minification, instead of whole file.

### CODE REVIEW

File: css_html_js_minify/html_minifier.py [hunk 1]
```diff
@@ -20,9 +20,22 @@ def condense_html_whitespace(html):
     '<i><b><a> test </a></b></i><br>'
     """  # first space between tags, then empty new lines and in-between.
     log.debug("Removing unnecessary HTML White Spaces and Empty New Lines.")
-    is_ok = "<textarea" not in html.lower() and "<pre" not in html.lower()
-    html = re.sub(r'>\s+<', '> <', html) if is_ok else html
-    return re.sub(r'\s{2,}|[\r\n]', ' ', html) if is_ok else html.strip()
+    #split the html on the <pre> and <textarea> tags, mantaining the separators
+    split = re.split('(<pre>|</pre>|<textarea>|</textarea>)', html)
```
Comment (juancarlospaco) [inline css_html_js_minify/html_minifier.py]: Possible problems with this:

```html
< pre >
<textarea id=foo >
</ textarea>
<!-- <pre> -->
```
You get the idea...

Comment (juancarlospaco) [inline css_html_js_minify/html_minifier.py]: I mean probably is better than the current one, but neither is a reliable solution yet...

Comment (bamless) [inline css_html_js_minify/html_minifier.py]: yeah, i managed to get a more relaiable solution, but it still can't handle comments right. At the end i decided to use this better version i wrote for my website and to pay atention not to put <pre...> or <textarea ...> in comments.

Comment (juancarlospaco) [inline css_html_js_minify/html_minifier.py]: I can re-open, merge, and open a new bug about the new details on the implementation.

Comment (bamless) [inline css_html_js_minify/html_minifier.py]: i'm sorry but i cannot reopen (deleted branch) :(. i've sent another pull request che that out
